packages/blocks/src/__internal__/rich-text/link-node/create-link.ts

```diff
@@ -45,7 +45,22 @@ export function createLink(page: Page) {
   // mock a selection style
   const range = getCurrentNativeRange();
   const rects = Array.from(range.getClientRects());
-  const mockSelection = new LinkMockSelection(rects);
+
+  const container = document.querySelector('.affine-editor-container');
```

Comment:
To prepare for the possibility of multiple editors in the future, you should use the method from 'query.ts'.

Reply:
I will update #1572 and #1561 later.